src/components/home/index.js

```diff
@@ -16,9 +21,35 @@ import {
 
 const Home = () => {
   const { showMemberRoleUpdateModal, isSuperUserMode } = userContext();
+  const {
+    state: { newMembers: newMembersList },
+  } = membersContext();
+  const { searchTerm } = searchMemberContext();
+  const filterMembersList = searchMembers(newMembersList, searchTerm);
+
   const { query } = useRouter() || { query: { dev: false } };
   const { dev } = query;
   const [isOptionKey, setIsOptionKey] = useState(false);
+  const [newMembers, setNewMembers] = useState([]);
+  const [scrollCount, setScrollCount] = useState(0);
+
+  const membersToShow = 10;
+
+  const fetchNewMembers = () => {
+    const newMemberArr = searchTerm ? filterMembersList : newMembersList;
+    const noOfMembersToShow = scrollCount * membersToShow;
+    const slicedNewMembersList = newMemberArr.slice(0, noOfMembersToShow);
+    setNewMembers(slicedNewMembersList);
+    setScrollCount(scrollCount + 1);
+  };
+
+  useEffect(() => {
+    fetchNewMembers();
+  }, [newMembersList]);
```

Comment:
we can add the `fetchNewMembers` function in the dependency array as pointed out in the warning by GitHub actions.

Reply:
Can you please tell me why we need to provide this function in the dependency array?

Reply:
Since the hook is dependent on the function, I thought we can add it. what do you think @sumitd94 @swarajpure ?